fluid_onnx/ops.py

```diff
@@ -224,8 +252,63 @@ def lppool_op():
     pass
 
 
-def matmul_op(inputs, attrs, outputs):
-    return make_node('MatMul', inputs=inputs, outputs=outputs)
+def mul_op(operator, scope):
+    inputs, attrs, outputs = get_op_io_info(operator)
+
+    # Flatten input(X) and input(Y) into 2-D matries
+    x_flat_out = [inputs['X'][0] + '@flatten_0']
```

Comment:
Please pay attention to the intermediate tensors n between nodes. They should be named uniquely in the ONNX graph.